tests/migration/paused.go

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package migration
+
+import (
+	"fmt"
+	"time"
+
+	kvconfig "kubevirt.io/kubevirt/tests/libkubevirt/config"
+	"kubevirt.io/kubevirt/tests/libmigration"
+
+	migrationsv1 "kubevirt.io/api/migrations/v1alpha1"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+
+	kvpointer "kubevirt.io/kubevirt/pkg/pointer"
+
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+
+	"kubevirt.io/kubevirt/tests/framework/checks"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"kubevirt.io/kubevirt/tests/libvmifact"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/libnet"
+	"kubevirt.io/kubevirt/tests/libvmops"
+)
+
+var _ = SIGMigrationDescribe("Live Migrate A Paused VMI", decorators.NoFlakeCheck, func() {
```

Comment:
I already regret creating this decorator :sweat_smile: 
Have we tried everything we could to make the flake-checker lane compatible with this?
I don't understand the error we're getting, do we know more about it?

Reply:
My bad! I got confused with the virt-controller log.
Actually, the issue is resources related:
```
"message": "0/2 nodes are available: 2 Insufficient memory. preemption: 0/2 nodes are available: 2 No preemption victims found for incoming pod."
```
The VMI is requesting `3Gi` and there are 2 `Entry`s that are running in parallel, and migrating:
3Gi x 2(running source virt-launcher) + 3Gi x 2(target virt-launcher).

Sorry for the wrong suggestion @vladikr. Why do we need 3Gi VMI, isn't 512Mi enough as for postcopy test `[test_id:4747]`?

Reply:
In the check-tests-for-flakes there are only 2 nodes (9Gi of memory for each one)

Reply:
> In the check-tests-for-flakes there are only 2 nodes (9Gi of memory for each one)

Unless one of the tests is decorated with `RequiresTwoWorkerNodesWithCPUManager`

Reply:
Hmm, none of the tests in the post copy file have `RequiresTwoWorkerNodesWithCPUManager`, yet `should make sure that VM restarts after failure` is working fine. 

I reduced the memory size of the VMI. It should work.

My main concern was that a smaller size VM was that it won't have enough time to spend migrating after it was paused. The only reason it didn't migrate in time was because of the pages it generated when it was active. 
But it looks like the test completes the fine with smaller values.

Reply:
Please remove `decorators.NoFlakeCheck`